### ISSUE TITLE
Remove tab order from help dialog header & body

### ISSUE THREAD

**Issue by allisonmuise** [Esri/crowdsource-reporter#381]
Title:
![11-14-2018 1-48-16 pm](https://user-images.githubusercontent.com/2752195/48514812-15f27000-e814-11e8-8933-022f32100dfa.png)
![11-14-2018 1-48-03 pm](https://user-images.githubusercontent.com/2752195/48514809-1559d980-e814-11e8-8411-3c32b922cee3.png)

Body:
![11-14-2018 1-48-35 pm](https://user-images.githubusercontent.com/2752195/48514810-15f27000-e814-11e8-857f-337a1cc58d68.png)
![11-14-2018 1-48-27 pm](https://user-images.githubusercontent.com/2752195/48514811-15f27000-e814-11e8-9fba-305c806eba67.png)

**Comment by ashishchoure** [comment]
@allisonmuise 

Looks like this issue is same as #389.

We will remove the tab index from title and body.
However, the same dialog is being used for share and geo-tagging functionality. Since these dialog's have some interactive contents, those will be focused. 

![image](https://user-images.githubusercontent.com/5525972/48554038-50870600-e903-11e8-87c3-e3fecbc08a01.png)

![image](https://user-images.githubusercontent.com/5525972/48554104-81673b00-e903-11e8-9886-713a06080edc.png)


Will this work?

**Comment by allisonmuise** [comment]
@ashishchoure yes - as you describe. remove the tab order from title and static content. Focus on interactive content is good. I missed testing the geotag window.
